Services/data/devicemodels/scripts/truck-02-state.js

```diff
@@ -17,8 +17,8 @@ var state = {
     longitude: center_longitude,
     speed: 80.0,
     speed_unit: "mph",
-    cargotemperature: 49.0,
-    cargotemperature_unit: "F"
+    temperature: 49.0,
```

Comment:
Updating the truck simulation to track 'temperature' instead of 'cargotemperature'. This was necessary now that the SetTemperature method is implemented (so that we can have this method write to one property, as opposed to trying to figure out if the caller wanted 'cargotemperature' or 'temperature' updated).